apps/web/src/components/forms/monitor/section-assertions.tsx

```diff
@@ -53,6 +55,63 @@ export function SectionAssertions({ form }: Props) {
 
   return (
     <div className="grid w-full gap-4">
+      <SectionHeader
+        title="Timing Setting"
+        description={
+          <>
+            Add specific time limits to your requests to receive notifications
+            if an endpoint takes longer than expected.
+          </>
+        }
+      />
+      <div className="grid w-full gap-4 sm:grid-cols-6">
+        <FormField
+          control={form.control}
+          name="degradedAfter"
+          render={({ field }) => (
+            <FormItem className="col-span-6 sm:col-span-3">
+              <FormLabel>Degraded</FormLabel>
+              <FormControl>
+                <Input
+                  className="bg-muted"
+                  type="number"
+                  placeholder="30"
+                  {...form.register("degradedAfter", {
+                    setValueAs: (v) => (v === "" ? null : Number.parseInt(v)),
```

Comment:
If we coerce the degradedAfter, we can return simply `v` not `Number.parseInt(v)` afaik.